src/main/java/de/dennisguse/opentracks/util/IntentUtils.java

```diff
@@ -99,6 +101,29 @@ public static Intent newShareFileIntent(Context context, long[] trackIds) {
                 .putExtra(Intent.EXTRA_TEXT, context.getString(R.string.share_track_share_file_body, trackDescription));
     }
 
+    /**
+     * Creates an intent to share a waypoint image with an app.
+     *
+     * @param context the context.
+     * @param uri     uri with the image to share.
+     */
+    public static Intent newShareImageIntent(Context context, Uri uri) {
+        String mime = context.getContentResolver().getType(uri);
+
+        // Because the #166 bug, when you import KMZ tracks then it creates file:/// from markers
```

Comment:
Could you use a line break after every sentence?
It is usually easier to read

Reply:
Of course ;)

Reply:
This is the last one (I promise): but could you modify the comment layout to use:
1. /* */
Because all 
2. Write one sentence per line followed by a line break
It is usually easier to read and will apply the proper formatting depending on the IDE's window width.